Packs/Tripwire/Integrations/Tripwire/Tripwire.yml

```diff
@@ -0,0 +1,532 @@
+category: Utilities
+commonfields:
+  id: Tripwire
+  version: -1
+configuration:
+  - defaultvalue: https://tripwire.com
+    display: Server URL (e.g. https://tripwire.com)
+    name: url
+    required: true
+    type: 0
+  - display: Username
+    name: credentials
+    required: false
+    type: 9
+  - display: Fetch incidents
+    name: isFetch
+    required: false
+    type: 8
+  - display: Incident type
+    name: incidentType
+    required: false
+    type: 13
+  - defaultvalue: '10'
+    display: Maximum number of incidents per fetch
+    name: max_fetch
+    required: false
+    type: 0
+  - defaultvalue: 3 days
+    display: First fetch time
+    name: first_fetch
+    required: false
+    type: 0
+  - additionalinfo: Rules IDs of elements versions to fetch, comma seperated.
+    display: Rule ids
+    hidden: false
+    name: rule_oids
+    required: false
+    type: 0
+  - additionalinfo: Nodes IDs of elements versions to fetch, comma seperated.
+    display: Node ids
+    hidden: false
+    name: node_oids
+    required: false
+    type: 0
+  - display: Trust any certificate (not secure)
+    name: insecure
+    required: false
+    type: 8
+  - display: Use system proxy settings
+    name: proxy
+    required: false
+    type: 8
+description: Tripwire is a file integrity management (FIM), FIM monitors files and folders on systems and is triggered when they have
+  changed.
+display: Tripwire
+name: Tripwire
+script:
+  commands:
+    - arguments:
+        - default: false
+          description: Versions IDs given comma seperated.
+          isArray: false
```

Comment:
```suggestion
          isArray: true
```

Reply:
is is not array in purpose, as this argument will be sent as a string to the api comma separated.
as a result I didnt think that there is a reason converting into array and then to string again,
So I would like to set it to string comma separated and send it directly to the api.

Reply:
I see. most of our customers are associating it differently, let's make sure we make it clear in the desc.